en/docs/develop-components/develop-integrations/develop-a-scheduled-integration.md

```diff
@@ -132,3 +132,18 @@ To visualize and monitor the performance of the scheduled integration component
  - The low-code diagram.
   
 To learn more about the observability details you can view via Choreo observability, see [Observability Overview](../../monitoring-and-insights/observability-overview.md).
+
+## Step 5: Monitoring Executions: Tracking Running and Completed Tasks
```

Comment:
Would be good to mention the component types that the execute view is applicable to and perhaps also mention how to access the Execute view. For example, 
"To track and monitor the executions associated with the deployed component, go to the left navigation menu and click **Execute**. "

Reply:
Updated